### ISSUE TITLE
Add support for paper waivers

### ISSUE THREAD

**Issue by rfdonnelly** [rfdonnelly/scma-waivers#1]
* [x] Create Google Spreadsheet
* [ ] Share it with maintainer of waivers
* [x] Pull waivers from Google Spreadsheet
* [x] Merge spreadsheet waivers with smartwaivers

**Comment by rfdonnelly** [comment]
Created gcp project `scma-smartwaiver` in cloud shell and added the Sheets API:

```console
$ gcloud projects create scma-smartwaiver
Create in progress for [https://cloudresourcemanager.googleapis.com/v1/projects/scma-smartwaiver].
Waiting for [operations/cp.5140338903501420306] to finish...done.                                                                                                                                                     
Enabling service [cloudapis.googleapis.com] on project [scma-smartwaiver]...
Operation "operations/acat.p2-425353441521-9d9ece6d-1df4-4609-ad14-8905b7a09138" finished successfully.
$ gcloud config set project scma-smartwaiver
Updated property [core/project].
rfdonnelly@cloudshell:~ (scma-smartwaiver)$ gcloud services enable sheets.googleapis.com
Operation "operations/acat.p2-425353441521-6a8e2366-4faf-4f59-aa77-19c16cbb5b7e" finished successfully.```

**Comment by rfdonnelly** [comment]
Added OAuth 2.0 client to the GCP scma-smartwaiver project with scopes:

* Google Sheets API ./auth/spreadsheets

**Comment by rfdonnelly** [comment]
Changed from personal oauth account to service account.  Service account owns the sheet.

Implemented in 8a7b65f973e2d5576c0201e2847aee2b7140dd4b
